games-strategy/crimson/crimson-0.5.3.recipe

```diff
@@ -0,0 +1,66 @@
+SUMMARY="A turn-based tactical war game"
+DESCRIPTION="The outcome of the war lies in your hands. You decide which \
+units are sent to the front lines, and when to unleash the reserves. Your \
+mission objectives range from defending strategically vital locations to \
+simply destroying all enemy forces in the area. Protect supply convoys or \
+raid enemy facilities to uncover technological secrets or fill your storage \
+bays so you can repair damaged units or build new ones in your own \
+factories. Lead your troops to victory! "
```

Comment:
I guess the white space between the exclamation mark and the double quotes is not needed ;-)